src/styles/blocs/navigation.scss

```
@@ -0,0 +1,57 @@
+.nav {
+
+  &__list {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+  }
+
+  &__link {
+    text-decoration: none;
+    font-size: 12px;
+    text-transform: uppercase;
+    display: flex;
+    align-items: center;
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    color: #060b35;
+    padding: 23px 0;
+
+    &:hover,
+    &:active {
+      color: #00acdc;
+    }
+
+  }
+
+  &__item {
+    position: relative;
+    margin-right: 20px;
+    &:last-child {
+      margin-right: 0;
+    }
+
+  }
+
+}
```

```suggestion
    }
  }
}
```
extra space in my opinion